sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -75,6 +60,8 @@ class CommunicationIdentifierModel(msrest.serialization.Model):
     :type is_anonymous: bool
```

Comment:
There is a typo in anonymous in line 59, and the id should say that it's the "full id".

Shouldn't this whole class be generated from the common swagger? Then it would have the right comments. Or is that different for Python?

Reply:
The generated file did not have some of the docstrings, therefore some of them are handwritten